### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by abhay-1403**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,3 +47,6 @@ and remotely.
 
 - Users should be able to provide feedback after using any facility.
 - Feedback submissions should be visible to the apartment manager/owner for review and improvement.
+
+
+# This is a test line
```
Comment (anikdas2204) [inline README.md]: Remove these and just add an extra line instead, so that i can test if my approval further carries the process as desired.

**Review by anikdas2204 — CHANGES_REQUESTED** [review]
Refer to the comment

File: README.md (hunk 1 above)
Comment (abhay-1403) [reply README.md:52]: but will that be considered as a change?

**Comment by anikdas2204** [comment]
This was done just to test a security feature.

**Pull request closed without merging**